examples/filter_5_string_bass_tabs.py

```diff
@@ -0,0 +1,28 @@
+import guitarpro
+import os
+
+def main(source):
+    print "Filtering...\n"
+    for dirpath, dirs, files in os.walk(source):
+    	for file in files:
+            try:
+                guitarProPath = os.path.join(dirpath, file)
+                tab = guitarpro.parse(guitarProPath)
+                for track in tab.tracks:
+                    if not track.isPercussionTrack:
+                        if len(track.strings) == 5:
+                            print guitarProPath
+            except:
+                pass
```

Comment:
This dirty `except: pass` is unrelated with PyGuitarPro. 
It is a fast workaround to avoid `GPException` when trying to parse a unsupported file (e.g. a `MastodonChords.txt` amongst guitar pro files). A cleaner way would be to use something like `if fnmatch.fnmatch(file, '*.gp*'):`.

Another mistake: in the `filter_trio_tabs.py` sample, the three variables (`drumsOK`, `bassOK`, `guitarOK`) should be declared first, not after `if(len(tab.tracks)) == 3:`. The script works though.

May I fix this ?